Release notes 2021-02 (#480)
SenseNet/sensenet.github.io: _updates/2021-02-22-february.md

---
title: February 2021
description: sensenet frontend updates of February 2021
---

## Valentine's Day 2021 💖💘

Love Is In The Air 🎵 

Everywhere I look around 💏

Love Is In The Air ☁️ 

Every sight and every sound.. 🎶

Another Valentine's day passed by, and we are here with the news. 
Check out what we done during these loving days. 

### admin-ui@1.17.0

> We completely renewed our search menu. Now we have advanced options for searching to make this feature easy to use. 💬 [#1180](https://github.com/SenseNet/sn-client/pull/1180)

Reviewer: ```suggestion
We completely renewed the search page. You have now advanced options for searching and filtering  the results to make this feature easy to use. 💬 [#1180](https://github.com/SenseNet/sn-client/pull/1180)
```

New options:  ⚡
- search by type (document, image, event, user, article, workspace, folder, task, memo, group)

> - search in dedicated container

Reviewer: ```suggestion
- search in a specific container
```

> - search for the contents what belongs to me

Reviewer: ```suggestion
- search for the contents what belongs to me (shared with me, assigned to me, etc.)
```

> - search by creation or modification date

Reviewer 2: filter instead of search?

> Note: This search input does not support query syntax, for queries use the command palette instead! 🌀

Reviewer: ```suggestion
Note: The advanced search input does not support query syntax, for working with queries use the command palette instead! 🌀
```

> Saving queries does not change, you can still store your search filters. 💾

Reviewer: ```suggestion
The save query feature did not change, you can still store your search filters. 💾
```

![Advanced search](/img/advanced_search.gif "Advanced search")

### @@sensenet/default-content-types@3.2.0

Default content types has been extended with some types 👽 [#1163](https://github.com/SenseNet/sn-client/pull/1163)

### @@sensenet/authentication-oidc-react@2.1.0

> We had to prepare in case the identity server shuts down 😎  [#1187](https://github.com/SenseNet/sn-client/pull/1187)

Reviewer: ```suggestion
We had prepare admin-ui to cases when the identity server is down 😎  [#1187](https://github.com/SenseNet/sn-client/pull/1187)
```

### @@sensenet/query@1.2.0

> Due to advanced search we also has some additions here 😵 [1180](https://github.com/SenseNet/sn-client/pull/1180)

Reviewer: ```suggestion
Due to the advanced search feature we also add some changes here 😵 [1180](https://github.com/SenseNet/sn-client/pull/1180)
```
